test/unit/generator/content/schedule/part3/get-top-up-summary.test.js

```diff
@@ -0,0 +1,35 @@
+const getTopUpSummary = require('../../../../../../app/generator/content/schedule/part3/get-top-up-summary')
+
+const mockAdjustment = {
+  currentValue: '1000.00',
+  newValue: '1200.00',
+  adjustmentValue: '200.00'
+}
+
+describe('Get top up summary', () => {
+  test('should return "The table below shows your revised payments." for key stack 6 entry', () => {

```

Comment:
update name

Reply:
`for key stack 1 entry' needs renamed

Reply:
Still not changed this test name

Reply:
This 1st test name is still saying the 6th key stack entry when the test itself is checking the 1st (`result.stack[0]`).

Please update it.

Reply:
updated